configs/python/backend/backend.yaml

```diff
@@ -39,7 +39,7 @@ scanners:
 #  'ScanBase64':
 #    - positive:
 #        flavors:
-#          - 'base64_file'
```

Comment:
`base64_file` isn't referenced anywhere besides the configs. In the scanner I use the flavor of just `base64` since it's not exactly a file. However I don't have a strong opinion and can change the flavor to `base64_file` if that's preferred.

Reply:
I'll review this scanner, appreciate the review here